src/v/cluster/controller.cc

```diff
@@ -763,6 +794,44 @@ controller::create_cluster(const bootstrap_cluster_cmd_data cmd_data) {
             co_return;
         }
 
+        // Check if there is any cluster metadata in the cloud.
+        auto bucket_opt = get_configured_bucket();
+        if (bucket_opt.has_value()) {
+            retry_chain_node retry_node(_as.local(), 300s, 5s);
```

Comment:
Oops! Started refactoring but scrapped it. This should be gated by the new config.

Reply:
Done